Lib/multiprocessing/pool.py

```diff
@@ -219,6 +232,23 @@ def __init__(self, processes=None, initializer=None, initargs=(),
                   self._result_handler, self._cache),
             exitpriority=15
             )
+        self._state = RUN
+
+    def __del__(self, _warn=warnings.warn):
```

Comment:
it's possible to call del with extra arguments ?

Reply:
No, you cannot. That's a trick to keep warnings.warn() alive during Python finalization. See other __del__() method of subprocess and asyncio module for example.